js/main.js

```diff
@@ -0,0 +1,174 @@
+// const $main = document.querySelector('main');
+const $breakfastRow = document.querySelector('#breakfast-row');
+const $lunchRow = document.querySelector('#lunch-row');
+const $dinnerRow = document.querySelector('#dinner-row');
+const $breakfastColumn = document.querySelector('#breakfast-column');
+const $lunchColumn = document.querySelector('#lunch-column');
+const $dinnerColumn = document.querySelector('#dinner-column');
+const $previewB = document.querySelector('#preview-b');
+const $previewL = document.querySelector('#preview-l');
+const $previewD = document.querySelector('#preview-d');
+const $breakfastUl = document.querySelector('#breakfast-list');
+const $lunchUl = document.querySelector('#lunch-list');
+const $dinnerUl = document.querySelector('#dinner-list');
+const $breakfastImage = document.querySelector('#breakfast-image');
+const $lunchImage = document.querySelector('#lunch-image');
+const $dinnerImage = document.querySelector('#dinner-image');
+const $breakfastText = document.querySelector('#breakfast-text');
+const $lunchText = document.querySelector('#lunch-text');
+const $dinnerText = document.querySelector('#dinner-text');
+// data.breakfastEntries.push({breakfastImage: xhrB.recipes.image, breakfastIngredients: )
+
+const obj = {
+  breakfastData: [],
+  lunchData: [],
+  dinnerData: [],
+  breakfastPreviewCounter: 0,
+  lunchPreviewCounter: 0,
+  dinnerPreviewCounter: 0
+};
```

Comment:
Do you need this & your `data` object? What's the difference between the two? It doesn't seem like you're using your `data` object anywhere

Reply:
I am not using it yet, was moreso setting up for local storage, to determine how many, and what recipes are saved. please advise.